packages/@aws-cdk/aws-rds/lib/instance-parameter-group.ts

```diff
@@ -0,0 +1,130 @@
+import { CfnOutput, Construct, IConstruct, Token } from '@aws-cdk/cdk';
+import { Parameters } from './props';
+import { CfnDBParameterGroup } from './rds.generated';
+
+/**
+ * A instance parameter group
+ */
+export interface IInstanceParameterGroup extends IConstruct {
+  /**
+   * Name of this parameter group
+   */
+  readonly parameterGroupName: string;
+
+  /**
+   * Export this parameter group
+   */
+  export(): InstanceParameterGroupImportProps;
+}
+
+/**
+ * Properties to reference a instance parameter group
+ */
+export interface InstanceParameterGroupImportProps {
+  readonly parameterGroupName: string;
+}
+
+/**
+ * Properties for a instance parameter group
+ */
+export interface InstanceParameterGroupProps {
+  /**
+   * Database family of this parameter group
+   */
+  family: string;
+
+  /**
+   * Description for this parameter group
+   */
+  description: string;
```

Comment:
Could this also be optional?

Reply:
No, it can't be optional.
refs. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-dbparametergroup.html#cfn-rds-dbparametergroup-description